src/Microsoft.DotNet.Build.Tasks/PrereleaseResolveNuGetPackageAssets.cs

```diff
@@ -8,11 +8,12 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Reflection.Metadata;
-using System.Reflection.PortableExecutable;
 using System.Text;
 using Newtonsoft.Json.Linq;
 using Newtonsoft.Json;
+using System.Security.Cryptography;
+using System.Reflection.PortableExecutable;
+using System.Reflection.Metadata;
 
 namespace Microsoft.DotNet.Build.Tasks
```

Comment:
I could rebase with an intermediate commit here, but I think it would be easier to just diff the two locally, you can see the original NuGet version here:

https://github.com/NuGet/NuGet.BuildTasks/blob/b8552204ff93245bb6e36ff9a4cb6091d05a7b94/src/Microsoft.NuGet.Build.Tasks/ResolveNuGetPackageAssets.cs